uml_lib/ebAPI_lib.py

```
@@ -67,58 +71,262 @@
+    final_meta = ResponseMeta(
+        href=href,
+        offset=current_offset,
+        limit=limit,
+        size=len(all_records),  # number of records actually aggregated
+        totalRecords=total_records,
+    )
+
+    # Return the structured response with the full dataset.
+    return ebResponse(query=query, meta=final_meta, records=all_records)
+
+def postTOAPI(URL, data, timeout: float = 30.0) -> Any:
```

Ruff (0.12.2)

Undefined name `Any`

(F821)